package.json

```diff
@@ -1,83 +1,97 @@
+  "license": "MIT",
+  "author": {
+    "email": "justin.s.dalrymple@gmail.com",
+    "name": "Justin Dalrymple"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/jdalrymple/node-gitlab"
   },
-  "keywords": [
-    "gitlab",
-    "api",
-    "es6",
-    "es5",
-    "request"
-  ],
+  "homepage": "https://github.com/jdalrymple/node-gitlab#readme",
   "bugs": {
     "url": "https://github.com/jdalrymple/node-gitlab/issues"
   },
-  "author": "Justin Dalrymple <justin.s.dalrymple@gmail.com>",
-  "license": "MIT",
-  "readmeFilename": "README.md",
+  "version": "4.2.0",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "browser": {
+    "dist/index.js": "dist/bundle.js",
+    "got": "ky"
+  },
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "npm run build:clean && npm run build:bundle",
+    "build:clean": "rimraf -rf dist && mkdirp dist",
+    "build:bundle": "rollup -c",
+    "commit": "npx git-cz",
+    "lint": "tslint 'src/**/*.{ts,tsx}' --format stylish --project tsconfig.json",
+    "prepublishOnly": "npm run build",
+    "prepush": "npm run lint",
+    "semantic-release": "semantic-release",
+    "test": "jest --debug --runInBand && codecov",
+    "test:bundles": "jest test/tests/bundles",
+    "test:infrastructure": "jest test/tests/infrastructure",
+    "test:services": "jest test/tests/services"
+  },
   "config": {
     "commitizen": {
       "path": "./node_modules/cz-conventional-changelog"
     }
   },
-  "files": [
-    "dist"
-  ],
+  "dependencies": {
+    "form-data": "^2.3.3",
+    "got": "^9.2.2",
+    "humps": "^2.0.1",
+    "ky": "^0.4.1",
+    "randomstring": "^1.1.5"
+  },
   "devDependencies": {
-    "@semantic-release/changelog": "^3.0.0",
-    "@semantic-release/git": "^7.0.4",
-    "@semantic-release/npm": "^5.0.4",
+    "@semantic-release/changelog": "^3.0.1",
+    "@semantic-release/git": "^7.0.5",
+    "@semantic-release/npm": "^5.0.5",
     "@types/humps": "^1.1.2",
     "@types/jest": "^23.3.5",
     "@types/parse-link-header": "^1.0.0",
     "@types/qs": "^6.5.1",
     "@types/request-promise": "^4.1.42",
     "@types/url-join": "^0.8.2",
     "@types/util.promisify": "^1.0.0",
+    "awesome-typescript-loader": "^5.2.1",
     "codecov": "^3.1.0",
     "cz-conventional-changelog": "^2.1.0",
-    "husky": "^1.1.1",
+    "husky": "^1.1.2",
     "jest": "^23.6.0",
     "jest-extended": "^0.11.0",
     "jest-tap-reporter": "^1.9.0",
-    "semantic-release": "^15.9.17",
+    "rollup": "^0.66.6",
+    "rollup-plugin-commonjs": "^9.2.0",
+    "rollup-plugin-json": "^3.1.0",
+    "rollup-plugin-node-resolve": "^3.4.0",
+    "rollup-plugin-terser": "^3.0.0",
+    "rollup-plugin-typescript2": "^0.17.1",
+    "semantic-release": "^15.10.3",
     "ts-jest": "^23.10.4",
     "tslint": "^5.11.0",
     "tslint-config-airbnb": "^5.11.0",
-    "typescript": "^3.1.2"
+    "tslint-config-prettier": "^1.15.0",
```

Comment:
thank you for adding prettier, would be great with prettier being a devDep too and include a npm script to check/fix formatting.